package_manager/contracts/PR/zeppelin/NewZosPackage.sol

```diff
@@ -0,0 +1,66 @@
+pragma solidity ^0.4.24;
+
+import '../RepoPackage.sol';
+import 'openzeppelin-solidity/contracts/ownership/Ownable.sol';
+import 'zos-lib/contracts/application/versioning/ImplementationProvider.sol';
+import 'zos-lib/contracts/application/versioning/ImplementationDirectory.sol';
+
+/**
+ * @title NewZosPackage
+ * @dev Collection of contracts grouped into versions.
+ * Contracts with the same name can have different implementation addresses in different versions.
+ */
+contract NewZosPackage is Ownable, RepoPackage {
```

Comment:
Should we start moving to RBAC?

Reply:
Agree but I think that's not part of the PoC, do you agree?